README.md

```diff
@@ -27,6 +27,10 @@ Crayfish contains the following services
 
 See the individual services for more information on their endpoints.
 
+## Security
+
+Crayfish microservices use JWTs to handle authentication like the rest of the Islandora CLAW.  It is disabled by default.  To enable, set `security enabled` to `true` in `cfg/cfg.php` for any microservice.  You can also set the path to an xml configuration fie for security a la [Syn][9] with the `security config` parameter.
```

Comment:
I wouldn't insist on a particular number, whatever works for the content is fine. I just try to avoid really, really long lines.